.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: Build and publish to ECR
+
+on:
+  workflow_dispatch:
+    branches:
+       -main
```

Comment:
Can you copy and paste **exactly** the file from another place e.g. https://github.com/alphagov/frontend/blob/main/.github/workflows/ci.yaml

I think the space between - and `main` has been eaten.